myia/compile/backends/python/python.py

```diff
@@ -297,17 +374,58 @@ def python_universe_getitem(c, universe, handle):
     operations.getitem: operator.getitem,
     operations.bool: operator.truth,
 }
+FUNCTION_MAP = {
+    operations.grad: op_grad,
+    operations.value_and_grad: op_value_and_grad,
+    operations.myia_next: myia_next,
+    operations.myia_iter: myia_iter,
+    operations.myia_hasnext: myia_hasnext,
+}
 
 
-def convert_operation(c, op, *inputs):
+def convert_operation(c, node, op, *inputs):
     """Convert an operation apply node to a Python code."""
     if op.value is operations.resolve:
         namespace = inputs[0].value
         name = inputs[1].value
         resolved = namespace[name]
+        code = None
+
         if resolved in OPERATION_MAP:
-            return f"operator.{OPERATION_MAP[resolved].__name__}"
-    raise NotImplementedError(f"Unsupported operation {op}")
+            # Resolve as Python's operator module function.
+            code = f"operator.{OPERATION_MAP[resolved].__name__}"
+        elif resolved in FUNCTION_MAP:
+            # Resolve as a specific function.
+            # Function will be available as global symbol in compiled function.
+            fn = FUNCTION_MAP[resolved]
+            code = c.register_global(fn.__name__, fn)
+        elif isinstance(resolved, Operation):
+            # Use operation's python implementation if available.
+            impl = resolved.defaults().get("python_implementation", None)
+            if impl is not None:
+                code = c.register_global(f"operation_{resolved.name}", impl)
+        elif isinstance(resolved, ModuleType):
+            # Module imported, probably to call an external function
+            # (e.g. `torch.argmax`)
+            code = c.register_global(resolved.__name__, resolved)
+        elif callable(resolved):
+            # Resolved is a callable.
+            # Callable will be available as global symbol in compiled function.
+            code = c.register_global(resolved.__name__, resolved)
```

Comment:
A callable can be any object that defines `__call__`. They may not have a `__name__`.